test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe(
+	'As a user I want the navigation block to fallback to the best possible default',
+	() => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'emptytheme' );
+		} );
+
+		test.beforeEach( async ( { admin } ) => {
+			await admin.createNewPost();
+		} );
+
+		test.afterAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyone' );
+		} );
+
+		test.afterEach( async ( { requestUtils } ) => {
+			await requestUtils.deleteAllPosts();
+		} );
+
+		test( 'default to a list of pages if there are no menus', async ( {
+			editor,
+			page,
+		} ) => {
+			await editor.insertBlock( { name: 'core/navigation' } );
+
+			// Necessary to wait for the block to be in loaded state else
+			// initial rendered state will not be resolved.
+			await page.waitForLoadState( 'networkidle' );
+
+			// Check Page List is in the list view.
+
+			// Open the list view.
+			await page.click( '[aria-label="Document Overview"i]' );
```

Comment:
Have you read the [best practices](https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/README.md#use-accessible-selectors) guide? We should favor role selectors wherever possible, it's more readable, accessible, and free of implementation details. The same goes for all the other selectors below.

Reply:
Ah `getByLabelText` should work